### PR TITLE
for AWS Aurora users and when using a DB Reader: new INI setting to enable aurora read replica read committed for fixing purge lag performance issue

### PR DESCRIPTION
**Pull request by justinvelluppillai**
### Description:

Allows enabling aurora read replica read committed

### Review

* [ ] Functional review done
* [ ] Potential edge cases thought about (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] Usability review done (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] Security review done [see checklist](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] Code review done
* [ ] Tests were added if useful/possible
* [ ] Reviewed for breaking changes
* [ ] Developer changelog updated if needed
* [ ] Documentation added if needed
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: core/Db.php [hunk 1]
```diff
@@ -184,6 +184,10 @@ public static function createReaderDatabaseObject($dbConfig = null)
 
         $db = @Adapter::factory($dbConfig['adapter'], $dbConfig);
 
+        if (!empty($dbConfig['aurora_readonly_read_committed'])) {
```
Comment (tsteur) [inline core/Db.php:187]: fyi we might still want to update `global.ini.php` here maybe @justinvelluppillai @diosmosis ?

Comment (diosmosis) [reply core/Db.php:187]: Do we want to document the setting?

Comment (tsteur) [reply core/Db.php:187]: I reckon it be good. Even if it's just useful mostly for us but then other devs will know what it's about.

Comment (justinvelluppillai) [reply core/Db.php:187]: I'll document and create a new pull request to add the global.ini.php update

**Pull request merged**